.drone.star

```diff
@@ -41,8 +45,8 @@ def main(ctx):
             "Ninja",
             trigger = build_trigger,
         ),
-        gui_tests(ctx, trigger = build_trigger, filterTags = ["@smokeTest"], version = "latest"),
-        gui_tests(ctx, trigger = build_trigger, depends_on = ["GUI-tests-@smokeTest"], filterTags = ["~@smokeTest"], version = "latest"),
+        gui_tests(ctx, trigger = build_trigger, version = "latest"),
+        #gui_tests(ctx, trigger = build_trigger, filterTags = ["@sslCertTest"], version = "latest"),
```

Comment:
Why did you comment this job? Do we need it when all the tests are run in earlier job?

Reply:
No we don't. I'll remove this in next commit.